charts/dev/opensearch/values.yaml

```
@@ -0,0 +1,151 @@
+opensearch-operator:
+  installCRDs: true
+
+clusterName: dev-opensearch 
```

This shouldn't be in this level of chart - since we can't `cp -r dev/opensearch staging/opensearch` with this in

Either we should leave it blank and fail the deployment if it's missing, or have something generic like `opensearch-cluster`

---

I will delete it to force people to provide it